cosmos/providers/dbt/parser/project.py

```diff
@@ -310,19 +291,78 @@ def _handle_config_file(self, path: Path) -> None:
                             for item in config_value
                             if item
                         ]
-
-            # dbt default ensures "materialized:view" is set for all models if nothing is specified so that it will
-            # work in a select/exclude list
-            config_types = [
-                selector_name
-                for selector in config_selectors
-                for selector_name in [selector.split(":")[0]]
-            ]
-            if "materialized" not in config_types:
-                config_selectors.append("materialized:view")
-
             # then, get the model and merge the configs
             model = self.models[model_name]
             model.config = model.config + DbtModelConfig(
                 config_selectors=set(config_selectors)
             )
+
+    def _handle_project_config(self, path: Path):
+        """Gets any configuration from project.yml"""
+        # parse the yml file
+        config_dict = yaml.safe_load(path.read_text())
+        # iterate over the models in the config
+        models = config_dict.get("models", {})
+        project_name = self.project_name
+        project = models.get(project_name)
+        if not project:
+            return
+        selectors = [f"+{selector}" for selector in SELECTORS]
+        for model_folder, configs in project.items():
+            self._handle_project_config_recursive(
+                configs,
+                selectors,
+                Path(self.dbt_root_path) / project_name / "models" / model_folder,
+            )
+
+    def _handle_project_config_recursive(
+        self, project_config: dict, selectors: list[str], project_path: Path
+    ) -> None:
+        """Goes bottom up to get tags against models (leaves to root).
+
+        This is because the most specific wins for some types.
```

Comment:
I'm not sure I understand what this comment means, do you mind explaining more/rewording?